general_tests/dispatcher_opts_it_test.go

```diff
@@ -114,14 +116,17 @@ func testDispatcherOptsAdminSetDispatcherProfile(t *testing.T) {
 			Tenant: "cgrates.org",
 			RemoteHost: &config.RemoteHost{
 				ID:              "SELF_ENGINE",
-				Address:         "*localhost",
+				Address:         "*internal",
```

Comment:
As discussed in the past, please keep here the localhost since we can see it better when troubleshooting.

Thanks!